img_checker.gemspec

```diff
@@ -9,4 +9,6 @@ Gem::Specification.new do |s|
   s.files       = ["lib/img_checker.rb"]
   s.homepage    = 'http://rubygems.org/gems/ImageSizeLinter'
   s.license     = 'GPL-3.0'
+  s.add_runtime_dependency "fastimage", [">= 2.0.0"]
+  s.add_development_dependency "fastimage", [">= 2.0.0"]
```

Comment:
Also it is a good thing to use [Ruby's pessimistic operator](https://robots.thoughtbot.com/rubys-pessimistic-operator) `~>` for mentioning gem versions. It's because running `bundle update` in any case ensures it doesn't include any breaking changes w.r.t the dependency. Read more [here](http://ryanbigg.com/2011/01/why-you-should-run-bundle-update). Though this is not that big deal here but it's always good to know :blush: